bin/release.py

```diff
@@ -213,10 +215,22 @@ def run():
 
     args = parser.parse_args()
 
-    # Let's make sure we're up-to-date and on master branch
+    github_project = args.github_project
+    github_user = args.github_user
+    main_branch = args.main_branch
+
+    if not github_project or not github_user or not main_branch:
+        print("main_branch, github_project, and github_user are required.")
+        print("Either set them in setup.cfg or specify them as command line arguments.")
+        return 1
+
+    # Let's make sure we're up-to-date and on main branch
     current_branch = check_output("git rev-parse --abbrev-ref HEAD")
-    if current_branch != "master":
-        print(f"Must be on the master branch to do this (not {current_branch})")
+    if current_branch != main_branch:
+        print(
+            f"Must be on the {main_branch} branch to do this;"
+            f" currently on {current_branch}"
+        )
```

Comment:
If you fix the flake8 configuration to ignore E501, then you don't have to do this.